app/src/main/java/com/example/fitjournal/library/presentation/screen/library/LibraryScreenViewModel.kt

```diff
@@ -21,11 +22,16 @@ class LibraryScreenViewModel @Inject constructor() : ViewModel() {
 
     init {
         // Dummy Data for now
-        val workoutList = listOf("Bench", "Squats", "Lateral Raises", "Elevated Goblet Squats")
-        val masterWorkoutList = workoutList.map { workout ->
-            LibraryWorkoutItem(
-                workoutName = workout,
-                workoutDetails = "Details for $workout go here"
+        val workoutMap = listOf("Bench", "Squats", "Lateral Raises", "Elevated Goblet Squats", "Burpees", "Bicep Curls", "Box Jumps", "Mountain Climbers", "Medicine Ball Slams", "Mountain Pose", "Modified Push-Ups", "Mason Twist", "Monkey Bars", "Modified Burpees", "Lunges", "Leg Press", "Lat Pulldowns", "Lying leg curls", "L-sit", "LandMine Twists").groupBy { it.first() }.toSortedMap()
```

Comment:
have the groupby and sorted map be a separate function so when we swap out local inputted data with db data it should be a quick fix